project/recursive_sorting.py

```diff
@@ -44,12 +44,47 @@ def merge_sort_in_place(arr, l, r):
 
 # TO-DO: implement the Quick Sort function below USING RECURSION
 def quick_sort( arr, low, high ):
+    # set a pivot
+    # pivot on the recursion, is the high on either side of the partition
+    # len(arr) - 1 is what we pass in to the arguments on the first call of the method
+    pivot = high
+
+    # go through each item. If it's less than. Move it to the left.
+    # if it's greater than. Move it to the right
+    flag = low
+    # this is the base case
+    if low < high:
+        for i in range(low, high - 1):
```

Comment:
if you remember from today's group coding, we used `range(0, high-1)` and that's where messed up. we should have used `range(low, high-1)` like you did here. nice going!